device/arista/x86_64-arista_7800r3a_36d2_lc/platform.json

```diff
@@ -0,0 +1,176 @@
+{
+    "chassis": {
+        "name": "7800R3A-36DM2-LC",
+        "components": [],
+        "fans": [],
+        "fan_drawers": [],
+        "psus": [],
+        "thermals": [
+            {
+                "name": "Cpu temp sensor"
```

Comment:
Could you set "controllable: false" for all thermals?
Some sonic-mgmt tests will fail otherwise.

Reply:
Hi @Staphylo, setting all thermals to false will make most thermal tests skipped, this means you don't support them?

Reply:
The `controllable: false` will just skip the 2 tests that try to call `Thermal.set_*_threshold()`
All the other thermal tests will be running the same.

Reply:
thanks, yes, it's the same as what I observed. 
I sent email for the `get_presence` and `get_high_critical_threshold` implementations.